ageWrapper shellcheck fail

```console
> agenix rekey
Collecting information about hosts. This may take a while...
error: builder for '/nix/store/2h1l7ahpvkf2gh5r7k4x8f0kvnflilb9-ageWrapper.drv' failed with exit code 1;
       last 7 log lines:
       >
       > In /nix/store/zi1zvbdnip8v6sbcrvqpfd385r78zxf7-ageWrapper/bin/ageWrapper line 22:
       > for file in /nix/store/kx8radkl1a86iz7i48vqhvaz4m5byav1-source/sec/age-yubikey-identity-7d5d5540.txt.pub; do
       >             ^-- SC2043 (warning): This loop will only ever run once. Bad quoting or missing glob/expansion?
       >
       > For more information:
       >   https://www.shellcheck.net/wiki/SC2043 -- This loop will only ever run once...
       For full logs, run 'nix log /nix/store/2h1l7ahpvkf2gh5r7k4x8f0kvnflilb9-ageWrapper.drv'.
error: 1 dependencies of derivation '/nix/store/ks5f95qblix7gspk0ic1r5m5gv2z1ra7-agenix-rekey.drv' failed to build
```

--- a/nix/lib.nix
+++ b/nix/lib.nix
@@ -67,13 +67,14 @@
       # For master identies with no explicit pubkey, try extracting a pubkey from the file first.
       # Collect final identity arguments for encryption in an array.
       masterIdentityArgs=()
-      # shellcheck disable=SC2041
-      for file in ${
+      masterIdentityArray=(${
         concatStringsSep " "
         (map
-          (x: "${escapeShellArg x.identity}")
+          (x: ''"${escapeShellArg x.identity}"'')
           (filter (x: x.pubkey == null) mergedMasterIdentities))
-      }; do
+      })
+      # shellcheck disable=SC2041
+      for file in "''${masterIdentityArray[@]}"; do
         # Keep track if a file was processed.
         file_processed=false
 

